prow/cluster/cluster.yaml

```diff
@@ -868,10 +868,10 @@ metadata:
   namespace: default
   name: "tide"
 ---
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
All prow jobs live in a single namespace (the same namespace as where tide lives). So I don't understand why this is necessary?

Reply:
the logs show tide failing to create pods in test-pods.
we're using clusterrole for our other components :man_shrugging:

Reply:
we could create roles for each namespace instead but i'm not sure what we intend to target.

Reply:
Tide doesn't create pods?

Reply:
ACK, logs read wrong, discussing in chat/